tests/src/python/CMakeLists.txt

```diff
@@ -2,6 +2,9 @@ set (ENABLE_MSSQLTEST FALSE CACHE BOOL "Enable MSSQL provider tests")
 set (ENABLE_ORACLETEST FALSE CACHE BOOL "Enable Oracle provider tests")
 set (ENABLE_HANATEST FALSE CACHE BOOL "Enable HANA provider tests")
 
+FILE(GLOB_RECURSE PythonTestsFiles "*.py")
+add_custom_target(python_tests SOURCES ${PythonTestsFiles})
```

Comment:
what about creating a custom target directly in [ADD_PYTHON_TEST](https://github.com/qgis/QGIS/blob/master/cmake/UsePythonTest.cmake)?
The command would something like `$PYTHONPATH $SOURCE`.
We would have an executable target directly accessible from the IDE.

Reply:
Yes that would be much better thanks